must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -60,9 +60,9 @@ done
 # Add important notfications to a notifications.txt file at the root of the noobaa collection path
 UNMANAGED_NOOBAA_NOTIF="The noobaa deployed on this cluster is not managed by the storagecluster CR and will not react to configuration changes made to storagecluster CR"
 
-NOOBAA_RECONCILE_STRATEGY=$( oc -n ${ns} get storagecluster -o jsonpath='{.items[0].spec.multiCloudGateway.reconcileStrategy}' 2> /dev/null)
+NOOBAA_RECONCILE_STRATEGY=$( oc -n "${ns}" get storagecluster -o jsonpath='{.items[0].spec.multiCloudGateway.reconcileStrategy}' 2> /dev/null)
```

Comment:
I might be mistaken but I believe this change will break the command

Reply:
I confirmed that the notifications file gets created with the appropriate text.

Reply:
Ok